src/gsy_e/models/market/day_ahead.py

```diff
@@ -0,0 +1,68 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from typing import Optional, TYPE_CHECKING
+
+from gsy_framework.constants_limits import ConstSettings, GlobalConfig
+from pendulum import DateTime, duration
+
+from gsy_e.gsy_e_core.blockchain_interface import NonBlockchainInterface
+from gsy_e.models.market import GridFee
+from gsy_e.models.market.future import FutureMarkets
+
+if TYPE_CHECKING:
+    from gsy_e.models.area.event_dispatcher import AreaDispatcher
+    from gsy_e.models.config import SimulationConfig
+
+DAY_AHEAD_MARKET_LENGTH_MINUTES = 60
```

Comment:
Yeah, I am not sure. I really wanted to hard code the value as this will probably never be changed (and also not confuse the tester and user). Curious what others think.

Reply:
actually, let me also move it to the ConstSettings. Thanks!